schedule/qam/15-SP6/mau-extratests-phub.yaml

```diff
@@ -7,6 +7,7 @@ schedule:
   - console/consoletest_setup
   - console/add_phub_extension
   - '{{sle15_x86_64}}'
+  - console/libjpeg_turbo
```

Comment:
IMO this should be part of general testing, not just SP6 MAU.

Reply:
According the feature: https://jira.suse.com/browse/PED-4889
Target Version/s: 15 SP6

Reply:
Which doesn't mean that it's SP6 specific. This module applies to SP7+ and Tumbleweed as well.

Reply:
Added job to SP7+ and Tumbleweed.  
Tumbleweed VR: https://openqa.opensuse.org/tests/4444987#step/libjpeg_turbo/10
For SP7 the packagehub is not ready, so the job failed: https://openqa.suse.de/tests/15304784#step/libjpeg_turbo/7